### ISSUE TITLE
Slow performances when sending string in flash memeory

### ISSUE THREAD

**Issue by bportaluri** [bportaluri/WiFiEsp#16]
WiFiEsp is slow when sending data using the F macro to store strings in flash memory.
```
client.print(F("<H1>WiFi ALA demo</H1>"));
```

Works fine when string is declared normally.
```
client.print("<H1>WiFi ALA demo</H1>");
```

**Comment by joysfera** [comment]
I have addressed this very issue in my fork of an ESP8266 library by providing a write() function that accepts the F() pointer. I think it would be neat to override the print() as well. I would suggest to address it here the same way.

**Comment by bportaluri** [comment]
Thank you.
Do you have a link to your code?

**Comment by joysfera** [comment]
https://github.com/joysfera/arduino-ESP8266

I have apparently provided "send()" and "print()" overloads. Well,
whatever the function name is, it works great.

**Comment by bportaluri** [comment]
Fixed in this commit: https://github.com/bportaluri/WiFiEsp/commit/f5c55db2193c79fbbdeedd308794ea46b1460497
Will be included in release 1.4.1
